### PR TITLE
Clean pyc files on clean install, and on reset.

### PR DESCRIPTION
**Pull request by JackMc**
Addresses #281

### CODE REVIEW

File: datacats/cli/create.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def reset(environment, opts):
     print 'Resetting...'
     environment.stop_supporting_containers()
     environment.stop_ckan()
+    clean_pyc(environment)
```
Comment (wardi) [inline datacats/cli/create.py]: reset affects the database, I don't see how cleaning the pyc applies

Comment (JackMc) [inline datacats/cli/create.py]: Denis wanted it because of CKAN upgrades.

Comment (wardi) [inline datacats/cli/create.py]: @deniszgonjanin don't you have to `install` when you upgrade ckan?

**Comment by JackMc** [comment]
Waiting until release-time for this to be merged.

**Pull request merged**